peps/pep-0736.rst

```diff
@@ -132,26 +158,35 @@ Prior Art
 =========
 
 Python already possesses a very similar feature in f-string interpolation where
-``f'{x=}'`` is effectively expanded to ``f'x={x}'`` [1]_.
+``f'{x=}'`` is effectively expanded to ``f'x={x}'`` (see
+`related GitHub issue <https://github.com/python/cpython/issues/80998>`__).
 
 Several modern languages provide similar features during function invocation,
 sometimes referred to as 'punning'. For example:
 
-* In Ruby,  ``f(x:, y:)`` is syntactic sugar for ``f(x: x, y: y)`` [2]_.
-* In ReasonML, ``f(~x, ~y)`` is syntactic sugar for ``f(~x=x, ~y=y)`` [3]_.
+* In Ruby,  ``f(x:, y:)`` is syntactic sugar for ``f(x: x, y: y)``. See the
+  `Ruby 3.1.0 release notes <https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/#:~:text=Other%20Notable%20New%20Features>`__ (search for "keyword arguments").
```

Comment:
But first: did we decide to replace these Chrome-specific links with regular anchor links supported by other browsers? 

https://github.com/python/peps/pull/3551#discussion_r1444184140

Reply:
I don't think those links are a problem to any browser, are they? Although they are a bit ugly.

Reply:
The links still load in other browsers, but at the top of the page. We can better serve everyone by linking to regular anchors.

However, looking again, this Ruby page doesn't have any anchors, and we do tell people to search for "keyword arguments", so let's just deal with the ugliness :)